docs/index.md

```diff
@@ -65,76 +69,76 @@ POST {json file} <url>/insert/
 }
 ```
 
 ### Response
 
 #### Body
 
-| Field            | Format       | Constraints                                                                     | Notes/Questions                                                                                  |
-|------------------|--------------|---------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| version          | Text         | "0.3.0"                                                                         | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
-| timestamp        | TimeDate     | yyyy-mm-ddThh:mm:ss.sss-hh:mm                                                   | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                               |
-| visit_date       | Date         | yyyy-mm-dd                                                                      | Effective date of score                                                                          |
-| alliance_id      | Text         | AlphaNumeric                                                                    | Return value provided by Feed                                                                    |
-| risk_score       | AlphaNumeric | 9.99                                                                            | Currently expecting numeric, future may be phrased by stating an odds ratio with the risk score. |
-| risk_score_notes | Text         | Additional notes (referral, remediation funds, etc to be returned to provider ) |                                                                                                  |
+| Field            | Format       | Constraints                       | Notes/Questions                                                                                  |
+|------------------|--------------|-----------------------------------|--------------------------------------------------------------------------------------------------|
+| version          | Text         | "0.3.0"                           | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
+| timestamp        | TimeDate     | yyyy-mm-dd hh:mm:ss.sss-hh:mm     | Current Timestamp (RFC 3339 complient date).                                                     |
+| visit_date       | Date         | yyyy-mm-dd                        | Effective date of score                                                                          |
+| patient_id       | Text         | AlphaNumeric                      | Same `patient_id` that was submitted to the API.                                                 |
+| risk_score       | AlphaNumeric | 9.99                              | Currently expecting numeric, future may be phrased by stating an odds ratio with the risk score. |
```

Comment:
In my mind, alphanumeric does not include a decimal point so I would call this Text.

Reply:
Agreed.